doc/start/quick-ceph-deploy.rst

```diff
@@ -25,181 +25,137 @@ are in this directory when executing ``ceph-deploy``.
    if you are logged in as a different user, because it will not issue ``sudo``
    commands needed on the remote host.
 
-.. topic:: Disable ``requiretty``
 
-   On some distributions (e.g., CentOS), you may receive an error while trying
-   to execute ``ceph-deploy`` commands. If ``requiretty`` is set
-   by default, disable it by executing ``sudo visudo`` and locate the
-   ``Defaults requiretty`` setting. Change it to ``Defaults:ceph !requiretty`` to
-   ensure that ``ceph-deploy`` can connect using the ``ceph`` user and execute
-   commands with ``sudo``.
-
-Create a Cluster
-================
+Starting over
+=============
 
 If at any point you run into trouble and you want to start over, execute
 the following to purge the Ceph packages, and erase all its data and configuration::
 
 	ceph-deploy purge {ceph-node} [{ceph-node}]
 	ceph-deploy purgedata {ceph-node} [{ceph-node}]
 	ceph-deploy forgetkeys
+	rm ceph.*
 
-If you execute ``purge``, you must re-install Ceph.
+If you execute ``purge``, you must re-install Ceph.  The last ``rm``
+command removes any files that were written out by ceph-deploy locally
+during a previous installation.
+
+
+Create a Cluster
+================
 
 On your admin node from the directory you created for holding your
 configuration details, perform the following steps using ``ceph-deploy``.
 
 #. Create the cluster. ::
 
-	ceph-deploy new {initial-monitor-node(s)}
+     ceph-deploy new {initial-monitor-node(s)}
 
    Specify node(s) as hostname, fqdn or hostname:fqdn. For example::
 
-	ceph-deploy new node1
-
-   Check the output of ``ceph-deploy`` with ``ls`` and ``cat`` in the current
-   directory. You should see a Ceph configuration file, a monitor secret
-   keyring, and a log file for the new cluster.  See `ceph-deploy new -h`_
-   for additional details.
-
-#. Change the default number of replicas in the Ceph configuration file from
-   ``3`` to ``2`` so that Ceph can achieve an ``active + clean`` state with
-   just two Ceph OSDs. Add the following line under the ``[global]`` section::
+     ceph-deploy new node1
 
-	osd pool default size = 2
+   Check the output of ``ceph-deploy`` with ``ls`` and ``cat`` in the
+   current directory. You should see a Ceph configuration file
+   (``ceph.conf``), a monitor secret keyring (``ceph.mon.keyring``),
+   and a log file for the new cluster.  See `ceph-deploy new -h`_ for
+   additional details.
 
 #. If you have more than one network interface, add the ``public network``
    setting under the ``[global]`` section of your Ceph configuration file.
    See the `Network Configuration Reference`_ for details. ::
 
-	public network = {ip-address}/{netmask}
-
-#. Install Ceph. ::
-
-	ceph-deploy install {ceph-node}[{ceph-node} ...]
+     public network = {ip-address}/{bits}
```

Comment:
i think of netmask a bitmask, e.g. '255.255.255.0'.  not sure what you call the trailing bit of a CIDR-style '1.2.3.4/24'.

https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing#CIDR_notation is sort of ambiguous.  maybe 'prefix bits'?